src/api/resources/base.py

```diff
@@ -530,7 +556,7 @@ class BaseApiViewSet(MethodAuthenticationMixin, viewsets.ModelViewSet, UpdatesMi
 
     pagination_class = StandardResultPagination
 
-    filter_backends = (DjangoFilterBackend, RelatedOrderingFilter, SearchFilter)
+    filter_backends = (DjangoFilterBackend, RelatedOrderingFilter, SafeSearchFilter)
```

Comment:
Are there any endpoints to worry about that somehow don't inherit from `BaseApiViewSet`?

Reply:
SearchFilter not being used anywhere else